CHANGELOG.md

```
@@ -1,93 +1,91 @@
 # Prowide Core - CHANGELOG
 
-#### 10.2.5 - December 2024
+#### 9.5.5 - December 2024
   * Update apache-commons-lang3 from 3.14.0 to 3.17.0 fixing derived apache-text dependency vulnerabilities 
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)